homeassistant/components/telegram_bot/strings.json

```
@@ -43,6 +43,10 @@
         "message_tag": {
           "name": "Message tag",
           "description": "Tag for sent message. In telegram_sent event data: {{trigger.event.data.message_tag}}."
+        },
+        "reply_to_message_id": {
+          "name": "Reply to message id",
+          "description": "Mark the message as a reply to a previous message. In telegram_callback handling, for example, you can use {{trigger.event.data.message.message_id}}"
```

This conflicts with placeholders. This is more fitting for the documentation anyways.

```suggestion
          "description": "Mark the message as a reply to a previous message."
```

---

understood!